awx/main/migrations/_create_system_jobs.py

```diff
@@ -0,0 +1,84 @@
+import random
+import logging
+
+from django.db import migrations, models
+from django.utils.timezone import now, timedelta
+
+logger = logging.getLogger('awx.main.migrations')
+
+__all__ = ['create_collection_jt', 'create_clearsessions_jt', 'create_cleartokens_jt']
+
+'''
+These methods are called by migrations to create various system job templates
+
+Create default system job templates if not present. Create default schedules
+only if new system job templates were created (i.e. new database).
+'''
+
+
+def create_clearsessions_jt(apps, schema_editor):
+
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')
+    Schedule = apps.get_model('main', 'Schedule')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)
+    now_dt = now()
+    random_time = now() + timedelta(minutes=random.randint(-30,30))
+    random_schedule_time = random_time.strftime('%Y%m%dT%H%M%SZ')
+
+    sjt, created = SystemJobTemplate.objects.get_or_create(
+        job_type='cleanup_sessions',
+        defaults=dict(
+            name='Cleanup Expired Sessions',
+            description='Cleans out expired browser sessions',
+            polymorphic_ctype=sjt_ct,
+            created=now_dt,
+            modified=now_dt,
+        ),
+    )
+    if created:
+        sched = Schedule(
+            name='Cleanup Expired Sessions',
+            rrule='DTSTART:%s RRULE:FREQ=WEEKLY;INTERVAL=1;COUNT=1' % random_schedule_time,
```

Comment:
just confirming I understand what these are doing: 

"if new system job was created, add a schedule for it that runs it weekly at around this time*"

* around this time == "some random time within + or - 30 mins of current time"

Reply:
@jakemcdermott correct.  I'll update that comment to be a little more clear.

Reply:
Also, this is really only _necessary_ for the collection system job (coming in a different PR).  I figured it couldn't hurt for these sys jobs as well, but there isn't a serious _need_ for the randomization here.

Reply:
I decided to keep it simple and take the randomization out of this PR and just keep it for the collection job for now.  Let me know if you think this should change.